### PR TITLE
Display flags and animations

### PR DESCRIPTION
**Pull request by sansaywm**
Screenshot_2020-09-25-01-23-58-379_com example firstproject
Screenshot_2020-09-25-01-24-02-235_com example firstproject
Screenshot_2020-09-25-01-24-06-658_com example firstproject

### CODE REVIEW

File: app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+        val animator = findViewById<ViewAnimator>(R.id.view_Animation)
+        val animationIn = AnimationUtils.loadAnimation(this, android.R.anim.fade_out)
+        val animationOut = AnimationUtils.loadAnimation(this, android.R.anim.fade_out)
+
+
+        animator.inAnimation = animationIn
+        animator.outAnimation = animationOut
+
+        val buttonPrev = findViewById<Button>(R.id.buttonPrev)
```
Comment (stasenkots) [inline app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt:25]: We don't need to use findViewById

File: app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        buttonPrev.setOnClickListener { animator.showPrevious() }
+
+        val buttonNext = findViewById<Button>(R.id.buttonNext)
```
Comment (stasenkots) [inline app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt:28]: We don't need to use findViewById

**Review by stasenkots — COMMENTED** [review]
Are your animation moving by itself or you must switch it?

**Comment by sansaywm** [comment]
I must switch it

**Comment by sansaywm** [comment]
I wanted to do it through a loop, but my phone crashed))

**Review by Vladislav7776 — APPROVED** [review]

File: app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt (hunk 2 above)
Comment (TMSlebedev) [reply app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt:28]: Please use synthetic

**Review by TMSlebedev — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt (hunk 2 above)
Comment (sansaywm) [reply app/src/main/java/com/example/firstproject/lesson2/Lesson2_animation.kt]: Thanks. Note has been corrected

**Review by TMSlebedev — APPROVED** [review]

**Pull request merged**